README.md

```diff
@@ -234,7 +320,7 @@ It is a low-overhead tracer with many important real-time compatible features:
 However, some settings need to be tuned for it to be fully real-time safe and for performance to be optimal for your use-case:
 
 * timers[^rt-1]: use [read timer](https://lttng.org/docs/v2.13/#doc-channel-read-timer) to avoid a write(2) call
-* sub-buffer[^rt-1] count and size:
```

Comment:
Why is this being removed?

Reply:
ah! I guess we can update this to reflect that we can configure it through the `Trace` action but not the `ros2 trace` command, but yeah we can just revert this change for now